contributors/devel/development.md

```diff
@@ -172,6 +172,13 @@ dependencies](godep.md).
 
 
 
+## Build with Bazel/Gazel
+
+Building with Bazel is currently experimental.  For more information,
+see [Build with Bazel].
```

Comment:
Yes, the link is defined at the end of the file.
Previewing on my fork looks right
![screen shot 2017-06-02 at 4 38 43 pm](https://cloud.githubusercontent.com/assets/14296719/26744003/0ab1c49c-47b2-11e7-8394-2c2d2d843a75.png)
and functions correctly.